app/models/talk/thumbnails.rb

```diff
@@ -20,7 +12,7 @@ def start_cues
   end
 
   def extracted?
-    talk.child_talks.map { |child_talk| child_talk.thumbnail_extractor.thumbnail_path.exist? }.reduce(:&)
+    talk.child_talks.map { |child_talk| child_talk.thumbnails.exist? }.reduce(:&)
```

Comment:
@marcoroth you probably want to update this too without the `exist?` delegation.

Reply:
Whoops, thanks!

Reply:
Fixed in https://github.com/adrienpoly/rubyvideo/commit/644c1347965f555977217db9c7c07582b1b9b8f3

Yeah, I don't love the solution/naming for the thumbnails right now. It's missing the whole logic for the thumbnails from YouTube/event fallbacks etc. But that's something to tackle in another PR.

Reply:
Ah I hadn't seen those!